unix/Makefile

```diff
@@ -29,6 +29,7 @@ tags:
 # Make a vanilla copy of micropython (unix) for use with upip and comparsion purposes
 tools:
 	cd $(PORT_TOP) && $(MAKE) clean deplibs all
+	pushd $(MPY_TOP)/mpy-cross && make && popd
```

Comment:
Included in make tools for completeness

Reply:
IIRC each command line of a makefile runs in a fresh shell, so no need to cd back. remove push/popd (like line above it)